components/layout_2020/flow/construct.rs

```diff
@@ -157,25 +157,29 @@ impl BlockContainer {
             builder.end_ongoing_inline_formatting_context();
         }
 
-        let mut contains_floats = builder.contains_floats;
+        type Intermediate<Node> = IntermediateBlockLevelBox<Node>;
+        #[derive(Default)]
+        struct Target {
```

Comment:
Why `Target`?

Reply:
That’s how it’s called in https://docs.rs/rayon_croissant/0.1.1/rayon_croissant/trait.ParallelIteratorExt.html#method.mapfold_reduce_into, and it’s a type that exists only for the purpose of calling that method.

Reply:
Maybe `Accumulator`? (Might work for `rayon_croissant` too)